src/Makefile.qt.include

```diff
@@ -314,8 +314,7 @@ BITCOIN_QT_CPP += \
   qt/walletview.cpp
 endif
 
-RES_IMAGES = \
-  qt/res/images/about.png
+RES_IMAGES = 
```

Comment:
I wasn't sure about that. Maybe we should keep the compile process open for further images.
That's why I left it. But I'm not sure.